bin/ch/ch.cpp

```diff
@@ -402,13 +405,23 @@ HRESULT CreateAndRunSerializedScript(const char* fileName, LPCSTR fileContents,
     IfJsErrorFailLog(ChakraRTInterface::JsGetCurrentContext(&current));
     IfJsErrorFailLog(ChakraRTInterface::JsSetCurrentContext(context));
 
+    // canonicalize that path name to lower case for the profile storage
```

Comment:
Considering we want it to work cross-platform, maybe we should consider not lowering the case for profile storage at all. Could end up with collisions on Unix.